pkg/restapi/edv/operation/operations.go

```diff
@@ -235,6 +249,23 @@ func (vc *VaultCollection) createDocument(vaultID string, document StructuredDoc
 	return vault.Put(document.ID, documentJSON)
 }
 
+// This function can't tell if the value before being encoded was precisely 128 bits long.
+// This is because the byte58.decode function returns an array of bytes, not just a string of bits.
+// So the closest I can do is see if the decoded byte array is 16 bytes long,
+// however this means that if the original value was 121 bits to 127 bits long it'll still be accepted.
```

Comment:
Should I worry about this case that I describe in the comment above this function?

Some context: this entire check is here just so that the server can let the client know when their provided value doesn't conform to the EDV spec.

Reply:
we don't usually work in bits, as long as you get 16 bytes array when decoding the id, we're ok

Reply:
Looks OK to me too.